docs/user-guide/src/SUMMARY.md

```diff
@@ -1,6 +1,6 @@
 # Metal3 Project
 
-<!-- markdownlint-disable no-empty-links -->
+[comment]: After adding the releasetag preprocessor the build fails with output "thread 'main' panicked at '', src/utils/fs.rs:45:10" if there are links with empty paths.
```

Comment:
In the case of mdbook Draft Chapters ([see point 6. in docs](https://rust-lang.github.io/mdBook/format/summary.html)) is a feature that is supposed to work. Draft Chapters are basically links that don't have a path and when the preprocessor was added those stopped working. I added the warning because the links are not working in this file as the mdbook documentation says the should. It's kind of an exception of an exception.

Reply:
I see! Thanks for the explanation!